app/admin/quests/create/page.tsx

```diff
@@ -0,0 +1,1049 @@
+                    label="End Date"
+                    placeholder="End Date"
+                    name="expiry"
+                  />
+                </div>
+              </div>
+              <div className="flex flex-row w-full gap-2 bg-gray-300 p-2 rounded-xl">
+                {CATEGORY_OPTIONS.map((category) => (
+                  <div
+                    onClick={() => {
+                      setQuestInput((prev) => ({ ...prev, category }));
+                    }}
+                    key={category}
+                    className="py-3 px-5 rounded-xl w-fit"
+                    style={{
+                      cursor: "pointer",
+                      backgroundColor:
+                        questInput.category === category
+                          ? "#ffffff"
+                          : "#29282B",
+                      color:
+                        questInput.category === category
+                          ? "#29282B"
+                          : "#ffffff",
+                    }}
+                  >
+                    <p className={styles.tagText}>{category}</p>
+                  </div>
+                ))}
+              </div>
+            </div>
+          </InputCard>
+
+          <div className="w-full items-center justify-center flex">
+            <div className="w-fit">
+              <Button
+                loading={buttonLoading}
+                disabled={isButtonDisabled}
+                onClick={() => handlePagination("Next")}
+              >
+                <p>Confirm Next</p>
+              </Button>
+            </div>
+          </div>
+        </>
+      ) : currentPage === 2 ? (
+        <>
+          <InputCard>
+            <div className="flex flex-col gap-8 w-full">
+              <p className={styles.cardHeading}>{headingText}</p>
+              <Textinput
+                onChange={(e) => {
+                  setNftUri((prev) => ({
+                    ...prev,
+                    name: e.target.value,
+                  }));
+                }}
+                value={nfturi.name}
+                name="rewards_nfts"
+                label="NFT Name"
+                placeholder="NFT Name"
+              />
+              <Textinput
+                onChange={handleQuestInputChange}
+                value={questInput.rewards_title}
+                name="rewards_title"
+                label="Rewards Title"
+                placeholder="NFT Name"
+              />
+              <Textinput
+                onChange={(e) => {
+                  setQuestInput((prev) => ({
+                    ...prev,
+                    rewards_img: e.target.value,
+                  }));
+                  setNftUri((prev) => ({
+                    ...prev,
+                    image: e.target.value,
+                  }));
+                  setBoostInput((prev) => ({
+                    ...prev,
+                    img_url: e.target.value,
+                  }));
+                }}
```

Comment:
Add a function here too long to have it inline

Reply:
done